sdk/core/core-http/package.json

```diff
@@ -38,25 +38,23 @@
     }
   },
   "files": [
-    "dist/**/*.node.js",
-    "dist/**/*.node.js.map",
+    "dist/",
```

Comment:
I am surprised that we don't include dist/index.js in the "files" section but somehow it is still packed.  Is it because of the "main" entry above?

Reply:
Strange